pkg/kubectl/cmd/apply_set_last_applied.go

```diff
@@ -112,7 +112,7 @@ func (o *SetLastAppliedOptions) Complete(f cmdutil.Factory, cmd *cobra.Command)
 	o.Codec = f.JSONEncoder()
 
 	var err error
-	o.Mapper, o.Typer = f.UnstructuredObject()
+	o.Mapper, o.Typer = f.Object()
```

Comment:
I don't have all the code in my head, but this leaps out as moving in the wrong direction.  Why the change?

Reply:
There is only Object() now.

Reply:
Everything is in the same path.  You ask the builder whether you want Unstructured() or Internal().  You can't opt out.